RS-Project10/Screen.java

```diff
@@ -0,0 +1,248 @@
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Integer> listOfParameters = new ArrayList<Integer>();
+        ShapesOperations operation = new ShapesOperations();
+        System.out.println("Enter origin x coordinate");
+        xCoordinate = sc.nextInt();
+        System.out.println("Enter origin y coordinate");
+        yCoordinate = sc.nextInt();
+        origin = new Point(xCoordinate, yCoordinate);
+        System.out.println("Enter point Opposite To Base x coordinate");
+        pointXCoordinate = sc.nextInt();
+        System.out.println("Enter point Opposite To Base y coordinate");
+        pointYCoordinate = sc.nextInt();
+        pointOppositeToBase = new Point(pointXCoordinate, pointYCoordinate);
+        System.out.println("Enter length of triangle");
+        length = sc.nextInt();
+        listOfParameters.add(0, length);
+        if (!operation.checkCoordinates(xCoordinate + length, yCoordinate + length) && 
+            !operation.checkCoordinates(xCoordinate, yCoordinate)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        if (!operation.checkCoordinates(pointXCoordinate, pointYCoordinate)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        listOfShapes.add(ShapesFactory.createTriangle("Triangle", origin, pointOppositeToBase, 
+            listOfParameters));
+    }
+
+    /**
+     * create circle objects
+     * @param list of shapes
+     */
+    public void addCircle(ArrayList<Shapes> listOfShapes) {
+        int radius;
+        Point origin;
+        int xCoordinate, yCoordinate;
+        Scanner sc = new Scanner(System.in);
+        ShapesOperations operation = new ShapesOperations();
+        ArrayList<Integer> listOfParameters = new ArrayList<Integer>();
+        System.out.println("Enter center x coordinate");
+        xCoordinate = sc.nextInt();
+        System.out.println("Enter center y coordinate");
+        yCoordinate = sc.nextInt();
+        origin = new Point(xCoordinate, yCoordinate);
+        System.out.println("Enter radius of circle");
+        radius = sc.nextInt();
+        if (!operation.checkCoordinates(xCoordinate + radius, yCoordinate + radius) && 
+            !operation.checkCoordinates(xCoordinate - radius, yCoordinate - radius)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        listOfParameters.add(0, radius);
+        listOfShapes.add(ShapesFactory.createShape("Circle", origin, listOfParameters));
+    }
+
+    public static void main(String[] args) {
+        int choice;
+        int option;
+        Scanner string = new Scanner(System.in);
+        Screen screen = new Screen();
+        ArrayList<Shapes> listOfShapes = new ArrayList<Shapes>();
+        ShapesOperations operation = new ShapesOperations();
+        do {
+            System.out.println("Graphics library");

```

Comment:
these lines can be more descriptive - like - Press 1 to add shapes to screen Or Press 2 to delete shapes from screen